spi_mem_manager/lib/spi/spi_mem_chip_arr.c

```diff
@@ -1397,4 +1397,6 @@ const SPIMemChip SPIMemChips[] = {
     {0xA1, 0x40, 0x16, "FM25Q32", 4194304, 256, SPIMemChipVendorFudan, SPIMemChipWriteModePage},
     {0xE0, 0x40, 0x14, "GT25Q80A", 1048576, 256, SPIMemChipVendorGenitop, SPIMemChipWriteModePage},
     {0xE0, 0x40, 0x13, "PN25F04A", 524288, 256, SPIMemChipVendorParagon, SPIMemChipWriteModePage},
-    {0x0B, 0x40, 0x18, "XT25F128B", 16777216, 256, SPIMemChipVendorXTX, SPIMemChipWriteModePage}};
+    {0x0B, 0x40, 0x18, "XT25F128B", 16777216, 256, SPIMemChipVendorXTX, SPIMemChipWriteModePage},
+    {0x20, 0x70, 0x17, "XM25QH64C", 8388608, 256, SPIMemChipVendorXMC, SPIMemChipWriteModePage},
+    {0x20, 0x70, 0x18, "XM25QH128A", 16776960, 256, SPIMemChipVendorXMC, SPIMemChipWriteModePage}};
```

Comment:
16776960 looks strange, where it came from?
<img width="319" alt="image" src="https://github.com/flipperdevices/flipperzero-good-faps/assets/1633132/4980813f-0023-4bf3-a669-ad5680e831ae">

Reply:
My mistake. Pasted wrong value, because I has calculate size as 65535 pages of 256 bytes. 655536 is correct of course. Already fixed.